sysdig/resource_sysdig_secure_custom_policy_test.go

```diff
@@ -25,13 +25,16 @@ func TestAccCustomPolicy(t *testing.T) {
 		},
 		Steps: []resource.TestStep{
 			{
-				Config: customPolicyWithName(rText()),
+				Config: customPolicyWithName(policy1),
 			},
 			{
 				ResourceName:      "sysdig_secure_custom_policy.sample",
 				ImportState:       true,
 				ImportStateVerify: true,
 			},
+			{
+				Config: customPolicyWithRulesOrderChange(policy1),
```

Comment:
Nit: can we keep this as `rText()` for consistency as all the other tests already use it? Or you would need to change all tests to use your new variable

Reply:
rText() generates a random string to make each policy different. The purpose of the test is to create a policy, modify it's rules order and not see drift on tf plan. Need the names to be the same to achieve this